UI/MemStickScreen.cpp

```diff
@@ -176,11 +177,18 @@ void MemStickScreen::CreateViews() {
 
 	if (!initialSetup_) {
 		rightColumnItems->Add(new Choice(di->T("Back")))->OnClick.Handle<UIScreen>(this, &UIScreen::OnBack);
+		rightColumnItems->Add(new Choice(di->T("Help")))->OnClick.Handle<MemStickScreen>(this, &MemStickScreen::OnHelp);
 	}
 
 	INFO_LOG(SYSTEM, "MemStickScreen: initialSetup=%d", (int)initialSetup_);
 }
 
+UI::EventReturn MemStickScreen::OnHelp(UI::EventParams &params) {
+	LaunchBrowser("https://ppsspp.org/guide_storage.html");
```

Comment:
nit: It's better for SEO if you use a consistent hostname (and scheme.)  Elsewhere www is used, and search engines consider them completely separate URLs - for example if 1000 people link to "https://ppsspp.org/" and 1000 people link to "https://www.ppsspp.org/", both sites are considered individually less popular than one that is linked to by 2000 people.  It's more complex than that, but at a basic level.

-[Unknown]